packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -8,5 +8,6 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
-  "cancel_sfdx_command_text": "SFDX: Cancel Active Command"
+  "cancel_sfdx_command_text": "SFDX: Cancel Active Command",
+  "force_apex_class_create": "SFDX: Create Apex Class"
```

Comment:
@ruthemmanuelle might already be out. But I think we should be OK using the same text we have from IDE 2 and prefixing with SFDX: .

So, @Sweetman please be sure to follow the text in IDE 2 for this and future commands.